app/assets/javascripts/transactions.js

```diff
@@ -23,4 +23,31 @@ $(function () {
     $('.transaction-type selected').html('Transaction');
     $('#new_transaction').show();
   });
+
+  $(document).on('change', '#transfer_bank_account_id', function(e) {
+    show_hide_exchange_rate();
+  });
+
+  $(document).on('change', '#transfer_reference_id', function(e) {
+    show_hide_exchange_rate();
+  });
+
 });
+
+function show_hide_exchange_rate() {
+  fromCurr = $('#transfer_bank_account_id option:selected').attr('data_currency');
+  toCurr = $('#transfer_reference_id option:selected').attr('data_currency')
+
+  if (fromCurr != undefined && toCurr != undefined && fromCurr != toCurr) {
+    $('#transfer_comission').parents('.col-sm-2').addClass('col-sm-1');
```

Comment:
@Gataniel let's use jQuery Chaining (http://www.w3schools.com/jquery/jquery_chaining.asp) here to avoid of selecting same elements a few times:
```
$('#transfer_comission').parents('.col-sm-2').addClass('col-sm-1').removeClass('col-sm-2');
```